connection.go

```diff
@@ -158,8 +158,9 @@ type Greeting struct {
 
 // Opts is a way to configure Connection
 type Opts struct {
-	// Timeout for any particular request. If Timeout is zero request, any
-	// request can be blocked infinitely.
+	// Timeout for response to a particular request. The timout is reset when
+	// push messages are received. If Timeout is zero request, any request can
```

Comment:
"If Timeout is zero request" -> "If Timeout is zero"?

Reply:
Thank you.
```patch
-       // Timeout for response to a particular request. The timout is reset when
-       // push messages are received. If Timeout is zero request, any request can
-       // be blocked infinitely.
+       // Timeout for response to a particular request. The timeout is reset when
+       // push messages are received. If Timeout is zero, any request can be
+       // blocked infinitely.
```